plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -252,10 +252,6 @@ class FossId internal constructor(
             pkg.vcsProcessed.revision.isEmpty() ->
                 "Package '${pkg.id.toCoordinates()}' has an empty VCS revision and cannot be scanned."
 
-            pkg.vcsProcessed.path.isNotEmpty() ->
```

Comment:
commit-msg: "[..] for browsing the code of a package [..]" To me it's unclear to what "browsing" refers to in context of the scanner. Would it make sense to mention this explicitly?

Reply:
I'm not certain how to phrase this in a better way (without going to broad either). It's about browsing the source code of a package in a third-party UI (similar to FOSSology) in order to create e.g. package configurations / license findings curation using a UI tool instead of writing YAML files.

Reply:
I've mentioned "third-party UI" as part of the commit message now, hopefully that makes it clearer.

Reply:
Thanks! It's clear to me now.